plugins/org.eclipse.thym.ui.importer/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure about the maven nature/builder.None of the other thym projects require maven to build on Eclipse IDE so m2e is not a requirement for Thym